IKU/IKU/Views/ViewModule/IKUCalendarView/IKUCalendarView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  IKUCalendarView.swift
+//  IKU
+//
+//  Created by Moon Jongseek on 2022/11/11.
+//
+
+import UIKit
+
+class IKUCalendarView: UIView {
+    // MARK: - Properties
+    var calendarView: CVCalendarView!
```

Comment:
p3

forced unwrapping은 최소화 하는게 좋지 않을까요?
init에서 따로 쓰지 않아도 좋을것 같아요

```suggestion
   var calendarView: CVCalendarView = CVCalendarView()
```

Reply:
[CocoaPods CVCalendar](https://cocoapods.org/pods/CVCalendar)
MIT License입니다!

Reply:
@sustainable-git 
P3 사항 반영되었습니다!